### DISCUSSION TITLE
[Q&A]: How to control extension when there is no .crx file

### DISCUSSION THREAD

**Discussion by kieuhuyhoang** [ultrafunkamsterdam/undetected-chromedriver#2055] [Q&A]
Hi everyone, I'm having the above problem as title:
+ I use the extension by Load Unpacked
+ And in Chrome's Default\Extensions there is no file containing the data of this extension.
+ Now I want to control that extension, what should I do?
+ And here is my silly code that doesn't work:

options = uc.ChromeOptions()
format_port = '127.0.0.1:{}'.format(2024)
options.debugger_address = format_port
// options.add_extension("AutoAddSubtitleYoutubeV11")
options.add_argument(f"--load-extension=./AutoAddSubtitleYoutubeV11./index.html")
options.add_argument("--window-position=-2400,-2400")
options.add_argument("--lang=en")
driver = uc.Chrome(options=options, headless=True)

// extension_page = "chrome://extensions/?id=fkegijkehhikejnokbmeijeabkfoipge/index.html"
// test = driver.get(extension_page)
print(driver.find_elements(By.TAG_NAME, "h3"))

Thanks everyone ♥

**Comment by PacharaXx** [comment]
`options.add_argument(f"--load-extension=./AutoAddSubtitleYoutubeV11./")`
only dir is it work?

**Reply by kieuhuyhoang** [reply]
I don't know how to use it, i find other solution using pyautogui
